frontend/src/components/TimerComponent/PopUp.js

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+const style = {
+  position: "absolute",
+  bottom: "-2%",
+  right: "0%",
+  width: "20%",
+  backgroundColor: "red",
+  border: "2px solid #000",
+  padding: "40px",
+};
+
+function PopUp({ seconds }) {
+  return (
+    <>
+      {seconds > 15 && <p style={style}>You have less than 30 seconds left!</p>}
+
```

Comment:
A fun way to organize these two conditional statements is with the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator).

e.g. `{seconds > 15 ? 1st p element : 2nd p element}`

or maybe because the p element and the style is the same, just the text inside a single p element:
`<p style={style}>{seconds > 15 ? "less than 30s left" : "less than 15s left"}</p>`

Basically a nice shorthand for an if statement in javascript 🙂